### PR TITLE
{vis}[system/system] Blender v3.0.0

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/Blender/Blender-3.0.0-binary.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'PackedBinary'
+
+name = 'Blender'
+version = '3.0.0'
+versionsuffix = '-binary'
```
Comment (boegel) [inline easybuild/easyconfigs/b/Blender/Blender-3.0.0-binary.eb:8]: Maybe `x64` is a better versionsuffix?

Comment (smoors) [reply easybuild/easyconfigs/b/Blender/Blender-3.0.0-binary.eb:8]: or even better: `-linux-x64` ?

Comment (smoors) [reply easybuild/easyconfigs/b/Blender/Blender-3.0.0-binary.eb]: @robert-mijakovic if you don't mind I went ahead and did the change for you.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node304.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/a88070ea92bfcf204a543284e803e8cc for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @robert-mijakovic!

File: easybuild/easyconfigs/b/Blender/Blender-3.0.0-binary.eb (hunk 1 above)
Comment (robert-mijakovic) [reply easybuild/easyconfigs/b/Blender/Blender-3.0.0-binary.eb]: @smoors Oh, I don't. I haven't even noticed the change before. Thank you.

**Pull request merged**